corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -373,6 +373,9 @@ function ComboboxEntry(question, options) {
                 matcher: function() {
                     return $input.val();
                 },
+                sorter: function(query, data) {
+                    return data;
+                },
```

Comment:
yeah that's what i had thought, but that wasn't the behavior i was seeing unfortunately